src/app.css

```
@@ -67,9 +68,45 @@
 	* {
-		@apply border-border;
+		border-color: var(--color-base-300);
 	}
+
 	body {
 		@apply bg-background text-foreground;
 	}
 }
+
+@plugin "daisyui/theme" {
+	name: 'forest';
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)